src/actions/actions.js

```diff
@@ -48,6 +53,10 @@ export const pushNewResource = newResource => ({
   newResource
 });
 
+// Handles the case where an existing resource is updated from the submission form
+export const UPDATE_EXISTING_RESOURCE = 'UPDATE_EXISTING_RESOURCE';
```

Comment:
The action type is not necessary anymore! You can use 
```js
updateExistingResource.type
```

Reply:
Fixed!